### PR TITLE
Korean: Update translation

### PR DESCRIPTION
**Pull request by telk5093**
Not fluently translated and seems that 7 strings are missed.
And it will be great that notice us when you add new strings in next update.

### CODE REVIEW

File: src/lang/korean.txt [hunk 1]
```diff
@@ -1730,6 +1736,8 @@ STR_CONFIG_SETTING_CITY_SIZE_MULTIPLIER                         :초기 대도
 STR_CONFIG_SETTING_CITY_SIZE_MULTIPLIER_HELPTEXT                :게임을 시작했을 때 일반 도시에 대한 대도시의 평균 크기를 설정합니다.
 STR_CONFIG_SETTING_RANDOM_ROAD_RECONSTRUCTION                   :지역 당국이 임의로 도로 보수공사를 시행할 확률: {STRING}
 STR_CONFIG_SETTING_RANDOM_ROAD_RECONSTRUCTION_HELPTEXT          :지역 당국이 임의로 도로 보수공사를 시행할 확률입니다. (0 = 하지 않음, 1000 = 최대)
+STR_CONFIG_SETTING_TOWN_MIN_DISTANCE                            :도시 사이의 최소 거리: {STRING}
+STR_CONFIG_SETTING_TOWN_MIN_DISTANCE_HELPTEXT                   :지도를 생성하고 무작위로 도시를 배치할 때 도시 사이의 최소 거리를 설정합니다.Set the minimum distance in tiles between towns for map generation and random founding
```
Comment (JGRennison) [inline src/lang/korean.txt]: Is this line intentional? There is some English text in the translation text?

**Comment by kiwitreekor** [comment]
I think it's his mistake...
He forgot to erase original text.

File: src/lang/korean.txt (hunk 1 above)
Comment (kiwitreekor) [inline src/lang/korean.txt]: I think it's his mistake...
He forgot to erase original text.

**Comment by telk5093** [comment]
Sorry I fotgot it and erased.

**Comment by JGRennison** [comment]
Thanks, this is merged now.

**Pull request merged**